web/components/ChatBox.js

```diff
@@ -1,86 +1,102 @@
 /**
- * Copyright (c) 2018, salesforce.com, inc.
+ * Copyright (c) 2020, salesforce.com, inc.
  * All rights reserved.
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or
  * https://opensource.org/licenses/BSD-3-Clause
  */
 
+import React, { useRef, useEffect } from 'react';
 import PropTypes from 'prop-types';
-
-const React = require('react');
 import './chat.css';
 const { env } = require('../../config');
 const { enableDropzone } = require('../../config')[env];
 
-class ChatBox extends React.Component {
-  componentDidUpdate(prevProps) {
-    if (prevProps.pendingMessage && this.props.currentText === '') {
-      document.getElementById('chat').innerText = '';
-    }
+const FOOTER_CLASS = 'slds-docked-composer__footer slds-grid slds-form' +
```

Comment:
if its only used in the one place and not changed is there any benefit to having this as a variable ?